index.js

```diff
@@ -0,0 +1,67 @@
+module.exports = {
+  "rules": {
+    "at-rule-empty-line-before": [ 2, "always", {
+      except: ["blockless-group"],
+    } ],
+    "at-rule-no-vendor-prefix": 0,
```

Comment:
> Yes I know, it was just to make it very visibile. 

Cool. Makes sense.

> Even if stylelint is use before cssnext, there is some prefix in our codebase

Oh I see. However, if no prefixes is the norm and the ones in `button.css` are the exception, it might be worth enabling the rule in the config and using [inline disabling](https://github.com/stylelint/stylelint#configuring-rules) where needed e.g.

```css
/* stylelint-disable property-no-vendor-prefix*/
.r-ResetButton {
  ...
}
/* stylelint-enable */
```